### PR TITLE
[Shop][Behat] Seeing order items on order summary

### PR DESCRIPTION
**Pull request by Zales0123**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Related tickets | 
| License         | MIT

Based on https://github.com/Sylius/Sylius/pull/5265.

### CODE REVIEW

File: features/checkout/seeing_order_summary/seeing_addresses_on_order_summary_page.feature [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    In order be certain about shipping and bulling address
+    As a Customer
+    I want to be able to see addresses on order summary page
+
+    Background:
+        Given the store operates on a single channel in "France"
+        And the store has a product "Targaryen T-Shirt" priced at "$19.99"
+        And the store ships everywhere for free
+        And the store allows paying offline
+        And I am logged in customer
```
Comment (tuka217) [inline features/checkout/seeing_order_summary/seeing_addresses_on_order_summary_page.feature]: `a logged in`

File: features/checkout/seeing_order_summary/seeing_addresses_on_order_summary_page.feature [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+        @todo
+        Scenario: Seeing the same shipping and billing address on order summary
+            Given I have product "PHP T-Shirt" in the cart
+            And I am at the checkout addressing step
+            When I specify the shipping address as "Ankh Morpork", "Frost Alley", "90210", "France" for "Jon Snow"
+            And I complete order with "Free" shipping method and "Offline" payment
+            Then I should be on the checkout summary page
```
Comment (tuka217) [inline features/checkout/seeing_order_summary/seeing_addresses_on_order_summary_page.feature]: Then I should see my order summary?

File: src/Sylius/Behat/Context/Ui/CheckoutContext.php [hunk 3]
```diff
@@ -450,6 +461,17 @@ public function iShouldSeeThisShippingAddressAsShippingAndBillingAddress(Address
     }
 
     /**
+     * @Given I complete order with :shippingMethod shipping method and :paymentMethod payment
+     */
+    public function iCompleteOrderWithShippingMethodAndPayment($shippingMethod, $paymentMethod)
```
Comment (tuka217) [inline src/Sylius/Behat/Context/Ui/CheckoutContext.php]: `$shippingMethodName` and `$paymentMethodName` ?

File: features/checkout/seeing_order_summary/seeing_items_on_order_summary_page.feature [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    I want to be able to see order items on the order summary page
+
+    Background:
+        Given the store operates on a single channel in "France"
+        And the store has a product "Lannister Coat" priced at "$19.99"
+        And the store has a product "Targaryen Jacket" priced at "$39.99"
+        And the store has a product "Stark T-Shirt" priced at "$21.50"
+        And the store ships everywhere for free
+        And the store allows paying offline
+        And I am logged in customer
```
Comment (tuka217) [inline features/checkout/seeing_order_summary/seeing_items_on_order_summary_page.feature]: `I am a logged`

**Comment by pjedrzejewski** [comment]
Closing in favor of #5324.

**Pull request closed without merging**